aws/README.md

```diff
@@ -27,4 +27,4 @@
    > Note: If you see Pending to the right of the notebook instance in the Status column, your notebook is still being created. The status will change to InService when the notebook is ready for use.
 
 3. Run Notebook
```

Comment:
Maybe note somewhere that it won't work in Studio currently?

Reply:
Done